unix/src/machine_stats/__init__.py

```diff
@@ -10,6 +10,14 @@
 
 from ansible.utils.path import unfrackpath
 
+# Setting default configuration parameters
+default_config = {
+    "ANSIBLE_HOST_KEY_CHECKING": False,
+    "ANSIBLE_GATHER_TIMEOUT": 180,
```

Comment:
I found the explanation [here](https://docs.ansible.com/ansible/2.9/reference_appendices/config.html#default-gather-timeout) useful.
I don't have much experience with Ansible, but any reason why we set the values to 180s to gather data about target nodes?

Reply:
We set it previously to fix some some issues with slow facts gathering: https://github.com/tidalmigrations/machine_stats/commit/ff4247fee1a5690c821e5d016a7630d5b8ddac6c